README.md

```diff
@@ -3,6 +3,9 @@ A bot for Public Lab
 
 plotsbot is an integrated system for bots across various interfaces, such as in an IRC chatroom, in GitHub issues, or on PublicLab.org. The bot consists of a set of behaviors, like "Greet" or "Help" (see below) which can work on one or more interface.
 
+**Where can I test it out?**  
+Currently, the bot lives in #publiclab-testing on OFTC or on #publiclab-testing on Matrix.
```

Comment:
Put `#publiclab-testing` in back ticks and link the Matrix url ( https://riot.im/app/#/room/#publiclab-testing:matrix.org ) in the second `#publiclab-testing` text.